test_suite/cloud/test_aws.py

```diff
@@ -343,6 +343,29 @@ def test_libc6_xen_conf_file_does_not_exist(self, host, rhel_only):
             file_to_check = '/etc/ld.so.conf.d/libc6-xen.conf'
             assert not host.file(file_to_check).exists, f'{file_to_check} should not be present in AMI'
 
+    def test_ena_support_correctly_set(self, host, instance_data_aws, rhel_only):
+        """
+        Check that Elastic Network Adapter support is enabled or disabled accordingly.
+        """
+        query_to_run = 'Reservations[].Instances[].EnaSupport'
+
+        command_to_run = [
+            'aws ec2 describe-instances',
```

Comment:
This is actually run locally in the node, where aws cli tool must be installed (otherwise, Terraform will not work).

To run a command locally in the node, `host.backend` module is used, and `run_local()` is called.

Reply:
ohhh, I see. Great